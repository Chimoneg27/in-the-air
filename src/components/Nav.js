import React from 'react';
import { NavLink } from 'react-router-dom';
import logo from '../assets/arrow-left.svg';
import '../Style/Navigation.css';

function Nav() {
  return (
    <nav>
      <div className="navigation">
        <img src={logo} alt="back" />
        <span>Back</span>
      </div>

      <ul className="Items">
        <li>
          <NavLink to="/air-details" activeClassName="active">
            Air Details
          </NavLink>
        </li>
      </ul>
    </nav>
  );
}

export default Nav;
