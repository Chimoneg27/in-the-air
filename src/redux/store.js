/* eslint-disable import/no-extraneous-dependencies */
import { configureStore } from '@reduxjs/toolkit';
import weatherReducer from './weatherSlice/weatherSlice';

const store = configureStore({
  reducer: {
    weather: weatherReducer,
  },
});

export default store;
