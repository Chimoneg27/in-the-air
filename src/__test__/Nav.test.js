import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Nav from '../components/Nav';

const testNav = (component) => render(
  <Provider store={store}>
    {component}
  </Provider>,
);

describe('Nav', () => {
  it('should render a header with a navigation bar and a banner', () => {
    testNav(<Nav />);
    expect(Nav).toMatchSnapshot();
  });
});
