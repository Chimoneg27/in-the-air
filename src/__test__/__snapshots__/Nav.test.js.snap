// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav should render a header with a navigation bar and a banner 1`] = `[Function]`;
